### PR TITLE
dhruv/TRAH-3834/Remove-JurisdictionScreens-MT5

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
## Changes:

Refactor the MT5 creation flow by removing Proof of Identity (POI), Proof of Address (POA), and Personal Details and Jurisdiction screens to streamline the process.

### CODE REVIEW

File: packages/cfd/src/Containers/__tests__/trading-password-manager.spec.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ jest.mock('@deriv/shared/src/services/ws-methods', () => ({
     default: 'mockedDefaultExport',
     WS: {
         verifyEmail: jest.fn(() => Promise.resolve()),
+        //abcdf
```
Comment (suisin-deriv) [inline packages/cfd/src/Containers/__tests__/trading-password-manager.spec.js]: why we added this comment 🙏

Comment (Dhruv-deriv) [reply packages/cfd/src/Containers/__tests__/trading-password-manager.spec.js]: was added for testing purpose have removed it 👍

**Review by suisin-deriv — APPROVED** [review]

**Review by utkarsha-deriv — APPROVED** [review]

File: packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx
Comment (amina-deriv) [inline packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx:1]: why do we remove this file?

File: packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-description.spec.tsx
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-description.spec.tsx:1]: why do we remove compare accountstable?

File: packages/appstore/src/components/modals/modal-manager.tsx [hunk 2]
```diff
@@ -297,9 +274,7 @@ const ModalManager = () => {
 
     return (
         <React.Fragment>
-            {is_jurisdiction_modal_visible && <JurisdictionModal openPasswordModal={openRealPasswordModal} />}
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/modal-manager.tsx:300]: we can remove these store values as well `is_jurisdiction_modal_visible`, `is_cfd_verification_modal_visible`

**Review by amina-deriv — CHANGES_REQUESTED** [review]
please remove corresponding store functions and css classes

File: packages/appstore/src/components/modals/modal-manager.tsx (hunk 2 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/modal-manager.tsx:300]: done

File: packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx
Comment (Dhruv-deriv) [reply packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx:1]: fixed

File: packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-title-icon.spec.tsx
Comment (amina-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-title-icon.spec.tsx:1]: why is this removed?

File: packages/cfd/src/Containers/props.types.ts [hunk 3]
```diff
@@ -328,7 +327,6 @@ export type TCFDDbviOnboardingProps = {
     fetchAccountSettings: () => void;
     has_created_account_for_selected_jurisdiction: boolean;
     has_submitted_cfd_personal_details: boolean;
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/props.types.ts]: i think we can remove this "has_submitted_cfd_personal_details". can you check this

File: packages/cfd/src/Containers/props.types.ts [hunk 4]
```diff
@@ -56,7 +56,6 @@ export type TMT5AccountOpeningRealFinancialStpModal = {
     enableApp: () => void;
     disableApp: () => void;
     toggleCFDVerificationModal: () => void;
```
Comment (amina-deriv) [inline packages/cfd/src/Containers/props.types.ts]: cant we remove this "toggleCFDVerificationModal". can you check this

File: packages/cfd/src/types/cfd-store.types.ts [hunk 5]
```diff
@@ -34,7 +34,6 @@ export type TCFDStore = {
     jurisdiction_selected_shortcode: string;
     toggleJurisdictionModal: () => void;
     has_submitted_cfd_personal_details: boolean;
```
Comment (amina-deriv) [inline packages/cfd/src/types/cfd-store.types.ts]: cant we remove these
```
 toggleJurisdictionModal: () => void;
 has_submitted_cfd_personal_details: boolean;
```

Comment (Dhruv-deriv) [reply packages/cfd/src/types/cfd-store.types.ts]: resolved

File: packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-title-icon.spec.tsx
Comment (Dhruv-deriv) [reply packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-title-icon.spec.tsx:1]: we can keep this as its not effecting anything

File: packages/cfd/src/Containers/props.types.ts (hunk 3 above)
Comment (Dhruv-deriv) [reply packages/cfd/src/Containers/props.types.ts]: this is not used anywhere so we can remove this

File: packages/cfd/src/Containers/props.types.ts (hunk 4 above)
Comment (Dhruv-deriv) [reply packages/cfd/src/Containers/props.types.ts]: we can remove this as well doesn't break anything

**Review by amina-deriv — APPROVED** [review]
LGTM
